source/plugin/items.rst

```diff
@@ -0,0 +1,57 @@
+==================
+Working with Items
+==================
+
+Items are represented through an ``ItemStack``. An ``ItemStack`` is an inventory item with information such as the amount of the item in the stack, the type of the item, and extra data such as durability. An ``Item`` itself is the graphical representation of an ``ItemStack`` as an entity.
+
+Checking an Item's Type
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Checking the type of the item is very simple. You just need to call the ``getItem()`` method in the ``ItemStack``.
+
+.. code-block:: java
+
+    import org.spongepowered.api.item.ItemType;
+    import org.spongepowered.api.item.ItemTypes;
+    import org.spongepowered.api.item.inventory.ItemStack;
+    
+    public boolean isStick(ItemStack stack) {
+        ItemType type = stack.getItem();
+        return type.equals(ItemTypes.STICK);
+    }
+    
+See how simple that is? Because sticks can stack, we can also find out how many are present.
+
+Getting the amount of items in an ``ItemStack`` is relatively easy. The ``getQuantity()`` method provided by ``ItemStack`` will handle this for us.
+
+Modifying ItemStack Data
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Manipulating data such as durability or the lore of an item is accomplished by simply using keys. You just need to specify the key that needs to be changed:
+
+.. code-block:: java
+    
+    import org.spongepowered.api.data.key.Keys;
+    import org.spongepowered.api.item.inventory.ItemStack;
+
+    public void setUnbreakable(ItemStack stack) {
+        stack.offer(Keys.ITEM_DURABILITY, -1);
```

Comment:
I checked before and didn't see it, so I assumed that -1 durability was unbreakable. Oh well, quick fix.